Source/API/EbApi.h

```diff
@@ -92,6 +92,13 @@ extern "C" {
 
 #define EB_BUFFERFLAG_EOS 0x00000001
 
+typedef enum EB_COLOR_FORMAT {
+    EB_YUV400,
+    EB_YUV420,
+    EB_YUV422,
+    EB_YUV444
+} EB_COLOR_FORMAT;
```

Comment:
better name with EB_CHROMAFORMAT_YUVXXX? 
typedef enum EB_COLOR_FORMAT  {
   EB_CHROMAFORMAT_YUV400 = 0,
  EB_CHROMAFORMAT_YUV_420        = 1,
  EB_CHROMAFORMAT_YUV_422        = 2,
  EB_CHROMAFORMAT_YUV_444        = 3,
  } EB_COLOR_FORMAT;

Reply:
it's not only about chroma, '4' denotes for luma actually